lib/modules/dosomething/dosomething_reportback/dosomething_reportback.install

```diff
@@ -228,6 +228,35 @@ function dosomething_reportback_schema() {
     ),
     'primary key' => array('rbid', 'fid'),
   );
+  $schema['dosomething_reportback_progress_log'] = array(
+    'description' => 'Log table of the total quantity of campaign nodes over time.',
+    'fields' => array(
+      'id' => array(
+        'description' => 'Primary key of log table.',
+        'type' => 'serial',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+      ),
+      'nid' => array(
+        'description' => 'The {node}.nid of the campaign.',
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+      'quantity' => array(
+        'type' => 'int',
+        'description' => 'The total quantity of approved reportbacks at this time.',
+        'not null' => TRUE,
+      ),
+      'timestamp' => array(
+        'description' => 'The Unix timestamp of the time this quantity was stored.',
+        'type' => 'int',
+        'not null' => TRUE,
+        'default' => 0,
+      ),
+    ),
+    'primary key' => array('id', 'nid'),
```

Comment:
They you should declare these a an index: https://api.drupal.org/api/drupal/includes!database!schema.inc/group/schemaapi, see `indexes`
Primary index is nor for search, but for unique identification of each row.